test/specs/scenarios/modal.js

```diff
@@ -27,6 +27,7 @@ export const modalScenarios = async(lang) => {
       }
       driver.sleep(500)
       welcome.clickOnOpenModalButton()
+      driver.sleep(500)
```

Comment:
I thought we wanted to move away from using `driver.sleep()` but I can still see `driver.sleep()` in this file. How about adding `await driver.wait(until.elementIsVisible(element))` in the `waitAndFindElement` before returning the element in [this file](test/utils/mochaw.js)?

Reply:
I thought we tried to add it in those tests and it didn't work hence we left these `sleep`s in place.

I will try the suggested approach.

Reply:
Done 
f68b7f0